react/ProductSummarySKUName.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+import React from 'react'
+// eslint-disable-next-line no-restricted-imports
+import { path } from 'ramda'
+import { useCssHandles } from 'vtex.css-handles'
+import { useProductSummary } from 'vtex.product-summary-context/ProductSummaryContext'
+
+const CSS_HANDLES = ['skuNameContainer']
+
+const ProductSummarySKUName = () => {
+  const { product } = useProductSummary()
+  const handles = useCssHandles(CSS_HANDLES)
+  const skuName: string = path(['sku', 'name'], product) ?? ''
```

Comment:
You can replace `path` with the optional operator `?.`

```suggestion
  const skuName: string = product?.sku?.name ?? ''
```